erts/preloaded/src/prim_zip.erl

```diff
@@ -350,7 +350,7 @@ prim_file_io({file_info, F}, _) ->
 	{error, E} -> throw(E)
     end;
 prim_file_io({open, FN, Opts}, _) ->
-    case ?CATCH prim_file:open(FN, Opts++[binary]) of
+    case ?CATCH(prim_file:open(FN, Opts++[binary])) of
```

Comment:
```suggestion
    case prim_file:open(FN, Opts++[binary]) of
```

As far as I understand, `prim_file:open/2` can't raise an exception. @jhogberg, do you agree, or could there be some other exception that the `try`/`catch` in `prim_file:open/2` won't catch?